hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/UserAgentSpanEnricher.java

```diff
@@ -30,7 +30,7 @@ public class UserAgentSpanEnricher extends AbstractTraceEnricher {
 
   private static final String CACHE_CONFIG_KEY = "cache";
   private static final String CACHE_CONFIG_MAX_SIZE = "maxSize";
-  private static final int CACHE_MAX_SIZE_DEFAULT = 10000;
+  private static final int CACHE_MAX_SIZE_DEFAULT = 20000;
```

Comment:
Where is the consumption to take from config ?

Reply:
This was already present in the code. Just added in conf files.